opennebula/shared_schemas.go

```diff
@@ -292,6 +294,10 @@ func generateVMTemplate(d *schema.ResourceData, tpl *vm.Template) {
 	for i := 0; i < len(disks); i++ {
 		diskconfig := disks[i].(map[string]interface{})
 
+		if diskconfig["image_id"].(int) < 0 {
+			break
```

Comment:
This would stop disk template building in the middle in case of this kind of disk declaration:
```
...
disk {
   image_id = X
}
disk {}
disk {
   image_id = X
}
...
```
in this case, first disk will be part of the template but not the last one

Reply:
you are right. 
I will fix this